angular/apis/etsy/gymnastics.js

```diff
@@ -0,0 +1,23 @@
+var data = require('./trending.json')
+var _ = require('lodash');
+var trending = data.results;
+
+
+function keyChecker(key){
+	return _.map(trending,key)
+};
+/* https://openapi.etsy.com/v2/taxonomy/seller/get?api_key=q4ubii6kukovuc0hl2e8myxx */
+/* https://openapi.etsy.com/v2/listings/trending?api_key=q4ubii6kukovuc0hl2e8myxx&fields=url,title,description, */
```

Comment:
Would prefer that these were documented better. Suggestions:

* Put the API key in a variable called `API` (so that it looks like a constant).
* Write a function that extracts some of the key fields and values from the `data` object and describes how they are used.
* Write a function that extracts the `results` and reduces the GIANT JSON OBJECT into some form that's easier to work with